builder/shell-for.nix

```diff
@@ -167,7 +167,9 @@ in
     buildInputs = systemInputs
       ++ mkDrvArgs.buildInputs or [];
     nativeBuildInputs = [ ghcEnv ]
-      ++ nativeBuildInputs
+      # We remove hsc2hs from dependencies provided by packages `build-tools`
+      # since it is already provided by `ghcEnv` (avoid collisions).
```

Comment:
This is safe because... hsc2hs comes with GHC, so we can't end up in a situation where packages depend on multiple different versions? or what?

Reply:
yes `hsc2hs` comes with GHC, and `ghcEnv` binaries are first in PATH so `hsc2hs` from ghcEnv has always been the one used anyway.